build.gradle

```diff
@@ -1,13 +1,11 @@
-import org.gradle.api.JavaVersion
```

Comment:
Is thiss not required for `sourceCompatibility`?

Reply:
green tick 🤷

Reply:
Can you spin it up locally just to make sure please?

Reply:
we do, in the integration tests.